DOCUMENTATION.md

```diff
@@ -29,6 +30,12 @@ This module implements all models for Resin Python SDK.
 ## Application
 
 This class implements application model for Resin Python SDK.
+
+Due to the API changes, the returned Application object schema changes as well. Here are the format of old and new returned object.
```

Comment:
nits:
 * change `Due to the API changes` to `Due to API changes`
 * change `changes as well` to `has changed`
 * change the last sentence to `Here are the formats of the old and new returned objects.`

EDIT: And obviously if you do make these tweaks, you should also make them in the other places this PR changes too ;-)